vaadin-date-range-picker/src/main/java/software/xdev/vaadin/daterange_picker/ui/DateRangePicker.java

```diff
@@ -352,6 +389,35 @@ public Div getOverlayContainer()
 		return this.overlayContainer;
 	}
 	
+	// -- LABELS --
+	
+	/**
+	 * Sets the label for the overlay Start-DatePicker
+	 * @param label
+	 */
+	public void setStartLabel(final String label)
+	{
+		this.getOverlay().getDpStart().setLabel(label);
+	}
+	
+	/**
+	 * Sets the label for the overlay End-DatePicker
+	 * @param label
+	 */
+	public void setEndLabel(final String label)
+	{
+		this.getOverlay().getDpEnd().setLabel(label);
+	}
+	
+	/**
+	 * Sets the label for the overlay DateRange-ComboBox
+	 * @param label
+	 */
+	public void setDateRangeOptionsLabel(final String label)
+	{
+		this.getOverlay().getCbDateRange().setLabel(label);
+	}
+	
```

Comment:
It would be great to have some _null_ checks here, what do you mean @AB-xdev

Reply:
@tboehm-xdev 
Sounds like a good idea 👍